calico-cloud_versioned_docs/version-3.15/variables.js

```diff
@@ -12,8 +12,8 @@ const variables = {
   rootDirWindows: 'C:\\TigeraCalico',
   nodecontainer: 'cnx-node',
   noderunning: 'calico-node',
-  cloudversion: 'v3.14.1-16',
-  clouddownloadurl: 'https://installer.calicocloud.io/manifests/v3.14.1-16',
+  cloudversion: 'v3.15.1-0',
+  clouddownloadurl: 'https://installer.calicocloud.io/manifests/v3.15.1-0',
```

Comment:
Does the `releaseTitle` variable needs to be updated here to `v3.15.1-0`?

Reply:
That's actually a relic from the other product docs. We don't use releaseTitle in CC.

Plans in the works for cleaning up variables.